### PR TITLE
Busco formigues

### PR DESCRIPTION
**Pull request by MarionaDSR**
Please review this, @casey-collab

### CODE REVIEW

File: catala/correr/correr.md [hunk 1]
```diff
@@ -3,3 +3,7 @@ La ostia que et fots és de campionat!
 Que no saps que les parets no poden ser de núvol de sucre?
 
 És clar que no, si no les formigues t'entrarien a casa ;-P
+
+Què fas?
+
+[Et poses a buscar formigues](formigues/formigues.md)
```
Comment (casey-collab) [inline catala/correr/correr.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: catala/correr/formigues.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Les petites formiguetes a l'estiu,
```
Comment (casey-collab) [inline catala/correr/formigues.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

Comment (casey-collab) [inline catala/correr/formigues.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by MarionaDSR** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :rabbit:

**Pull request merged**